data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,655 @@ mission "Remnant: Face to Maw 2B"
+	on complete
+		event "korath hunting"
+		conversation
+			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter..." he is interrupted by a flashing light on his commlink. He briefly reads something, then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defence prefect. Chilia has requested that you meet with him on Caelian immediately."`
+
+
+
+event "korath hunting"
+	government "Korath"
+		"attitude toward"
+			"Indigenous Lifeform" -.01
+			"Ember Waste" -.01
+
+
+
+mission "Remnant: Cognizance 3"
+	name "Search for Korath"
+	description "Prefect Chilia has asked you to track down a Korath fleet"
+	source "Caelian"
+	landing
+	waypoint
+		system "Aescolanus"
+	to offer
+		has "Remnant: Cognizance 2: done"
+	on offer
+		log "People" "Chilia" `Prefect Chilia is, compared to most military leaders you have met, an unusual person. He is inclined to do things himself, rarely standing on protocol, if at all.`
+		"remnant chilia" ++
+		conversation
+			branch chilia
+				"remnant chilia" > 1
+			`The <ship> has barely come to a rest on its designated pad when your panel is blinking with a request for permission to come aboard. A glance at the viewscreen shows a fit young man with a military bearing standing at your airlock. You quickly check with the starport control and confirm that the prefect in charge of Remnant military activities is indeed waiting to board your ship. You hit the unlock button and watch as Prefect Chilia strides onboard.`
+			`	"Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles deprecatingly. "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them.`
+				goto main
+			label chilia
+			`The <ship> has barely come to a rest on its designated pad when your panel is blinking with a request for permission to come aboard. A quick glance at the viewscreen shows Prefect Chilia standing at your airlock. You quickly hit the unlock button and watch as he strides onboard.`
+			`	"Greetings, Captain <first>," he sings. "You seem to have a knack for finding things of importance to us. I am glad, however, that you and Plume discovered this."`
+			label main
+			branch voidrifle
+				has "Remnant: Face to Maw 2B: done"
+			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites, and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. There are military implications of the Korath harvesting the void sprites, and we must take steps to prevent this from happening.`
```

Comment:
```suggestion
			`	His gestures take on an abrupt cadence without many of the normal connecting movements that you are accustomed to. "Now, for the task at hand: It seems the Korath have discovered the void sprites and shot at enough of them that almost all have retreated into the depths of Nasqueron and Slylandro. There are military implications of the Korath harvesting the void sprites, and we must take steps to prevent this from happening.`
```